cassiopeia/data/CassiopeiaTree.py

```diff
@@ -1401,7 +1401,11 @@ def get_mutations_along_edge(
 
         mutations = []
         for i in range(self.n_character):
-            if parent_states[i] != child_states[i]:
+            
```

Comment:
It is possible for a parent state to be ambiguous if _all_ of its children have the same ambiguous state. 

But you are right -- if a parent has an ambiguous state, the child must have the same ambiguous state. If a child does _not_ have an ambiguous state, that reflects on the true state of the parent (either 0, if the child's state does not appear in the other child's ambiguous states, or it will resolve the ambiguous state to a particular state `s`).

Reply:
Done.